app/src/state/ducks/auth.js

```diff
@@ -211,3 +211,14 @@ export function updatePassword(password, newPassword) {
     }
   };
 }
+
+// Selectors
+export const authState = state => state.auth || {};
+export const getMe = createSelector(
+  authState,
+  state => state
+);
```

Comment:
You might want to change getMe to:

`createSelector(state, state => state.auth.me)`

Then isLoggedIn becomes

`isLoggedIn = state => getMe(state) !== null ? true : false`